helm/superset/templates/init-job.yaml

```diff
@@ -24,6 +24,7 @@ metadata:
     "helm.sh/hook-delete-policy": "before-hook-creation"
   namespace: {{ .Release.Namespace }}
 spec:
+  ttlSecondsAfterFinished: 0
```

Comment:
Can you make this optional? We already have an annotation that controls the cleanup via Helm: 

```yaml
"helm.sh/hook-delete-policy": "before-hook-creation"
```

I've found it helpful to keep the completed job around after running in order to see logs, etc. What's worse, most log aggregators like DataDog won't scrape logs of a given container until it's been running for 30s, so if the thing goes away immediately, logs won't be viewable.

**Suggestion:**
```yaml
# vaules.yaml
init:
  ttlSecondsAfterFinished: ~
```


```
{{- if .Values.init.ttlSecondsAfterFinished }}
ttlSecondsAfterFinished: {{ .Values.init.ttlSecondsAfterFinished }}
{{- end }}
```

Reply:
Good call, that way in preview/dev environment we can still troubleshoot and production we can reduce pressure on api server.